sequencer/api/merklized_state.toml

```diff
@@ -0,0 +1,4 @@
+[route.getfeebalance]
+PATH = ["/fee_balance/:address"]
+":address" = "Literal"
+DOC = "Get current balance in address account."
```

Comment:
"Balance in fee state" or something like that?

Reply:
And mention that we are expecting an ethereum address in hex format.

Reply:
done in d19a139fb2599f6147bd6d5413adc5f0891338e3